apps/vmq_server/src/vmq_cluster.erl

```diff
@@ -169,12 +176,32 @@ check_ready([Node|Rest], Acc) ->
     ok = vmq_cluster_node_sup:ensure_cluster_node(Node),
     %% We should only say we're ready if we've established a
     %% connection to the remote node.
-    IsReady1 = IsReady andalso vmq_cluster_node_sup:is_reachable(Node),
+    S = vmq_cluster_node_sup:node_status(Node),
```

Comment:
Can this be named Status? I think will be more readable for contributors

Reply:
Sure! :+1: